zerver/lib/markdown/__init__.py

```diff
@@ -1826,17 +1845,27 @@ def handleMatch(self, m: Match[str]) -> Optional[Element]:
                 el.set("class", "user-mention")
                 text = f"@{text}"
             el.text = markdown.util.AtomicString(text)
-            return el
-        return None
-
+            return el, match_span[0], match_span[1]
```

Comment:
`return el, m.start(), m.end()` seems a little cleaner and avoids the separate `span()` call.

Reply:
made these changes.